exampleCustomAudioDevice/build.gradle

```diff
@@ -0,0 +1,76 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+
+    lintOptions {
+        warningsAsErrors true
+        disable "GradleDependency", "UnusedResources"
+    }
+
+    defaultConfig {
+        applicationId "com.twilio.video.quickstart"
```

Comment:
```suggestion
        applicationId "com.twilio.video.quickstart.customaudiodevice"
```

Reply:
Application id updated to `com.twilio.video.examples.customaudiodevice`